### PR TITLE
wbooks role created in postgresql. Test folder added to .gitignore

### PR DESCRIPTION
**Pull request by jhonmunoz**

### CODE REVIEW

File: wBoooksApi/.idea/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hdf1996) [inline wBoooksApi/.idea/.generators:1]: Can we add the entire .idea folder to the git ignore? Remember to remove these files after that

File: wBoooksApi/app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
+class PostsController < ApplicationController
```
Comment (hdf1996) [inline wBoooksApi/app/controllers/posts_controller.rb:1]: Hmm why did you add this controller? We aren't using the Post controller or model at all

Comment (hdf1996) [reply wBoooksApi/app/controllers/posts_controller.rb:1]: By the way, we try to not use the scaffold if possible, it creates a lot of things that probably you aren't using

File: wBoooksApi/app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <td><%= link_to 'Edit', edit_post_path(post) %></td>
+        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<br>
+
+<%= link_to 'New Post', new_post_path %>
```
Comment (hdf1996) [inline wBoooksApi/app/views/posts/index.html.erb:29]: Idem with the views

File: wBoooksApi/db/migrate/20180611203152_create_posts.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
```
Comment (hdf1996) [inline wBoooksApi/db/migrate/20180611203152_create_posts.rb:1]: You'll need to remove this migration too. After that you will need to do rails db:drop && rm db/schema.rb && rails db:create rails db:migrate

**Review by hdf1996 — APPROVED** [review]

File: wBoooksApi/app/controllers/posts_controller.rb (hunk 2 above)
Comment (jhonmunoz) [reply wBoooksApi/app/controllers/posts_controller.rb:1]: I ran an scaffold as indicated in the guide provided in the link of the training. Obviously didnt have to do this step

File: wBoooksApi/app/views/posts/index.html.erb (hunk 3 above)
Comment (jhonmunoz) [reply wBoooksApi/app/views/posts/index.html.erb:29]: deleting
